[webkitcorepy] six is part of the Python standard library

JonWBedard opened the pull request:
#### 55f4a9ce14e9eba0a6894b3b67a32c644be95858
<pre>
[webkitcorepy] six is part of the Python standard library
<a href="https://bugs.webkit.org/show_bug.cgi?id=232625">https://bugs.webkit.org/show_bug.cgi?id=232625</a>
&lt;rdar://problem/84931857 &gt;

Reviewed by NOBODY (OOPS!).

* Scripts/libraries/webkitcorepy/setup.py: Exclude six, bump version.
* Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py: Ditto.
</pre>

overclockedllama commented:
https://bugs.webkit.org/show_bug.cgi?id=173801

rniwa commented:
Hi, thanks for posting a patch but we don't accept Github PR merges since this Github instance is a mirror of git.webkit.org, which in turn is a mirror of svn.webkit.org. We can continue the discussions & review of the code in the Bugzilla bug you opened.

JonWBedard committed:
[webkitcorepy] six is part of the Python standard library
https://bugs.webkit.org/show_bug.cgi?id=232625
<rdar://problem/84931857>

Reviewed by NOBODY (OOPS!).

* Scripts/libraries/webkitcorepy/setup.py: Exclude six, bump version.
* Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py: Ditto.
